### PR TITLE
Keep custom lms version for github packages gem

### PR DESCRIPTION
**Pull request by justincaldwell**
This explicitly creates a lms version (3.1.0.lms) since it's different than master 3.1.0. Merging will effect the current lms setup as it will change the version in the Gemfile.lock on lms.

### CODE REVIEW

**Comment by kidsalsa** [comment]
@przemyslaws-iRonin @szysza do you see any issues with merging this? It relates to this PR https://github.com/acima-credit/lease_management_system/pull/4639 and must be merged before it. It looks fine to me, but I wanted to get a second pair of eyes on it before I do the merge.

**Review by szysza — APPROVED** [review]
Looks good to me.

**Review by przemyslaws-iRonin — APPROVED** [review]
LGTM

I'm just wondering what was a reason for [this](https://github.com/acima-credit/attr_encrypted/commit/6e4002d218ed5d55793fa7fd11016ccca3258e27) release which makes a difference between normal and lms versions. 

Was it the fault of the older rails version in LMS or something other? Maybe we don't need it now?

**Comment by kidsalsa** [comment]
@przemyslaws-iRonin I don't remember the issue that caused me to make that change a few years ago. I Googled around to see if I could find it again with no luck. I'll merge this PR and then look into whether we can go back to the non-custom version of attr_encrypted in this ticket: https://acima.atlassian.net/browse/LMS-12169

**Pull request merged**
